llvm/test/Transforms/MemProfContextDisambiguation/basic.ll

```diff
@@ -105,9 +106,9 @@ attributes #6 = { builtin }
 !0 = !{i64 8632435727821051414}
 !1 = !{i64 -3421689549917153178}
 !2 = !{!3, !5}
-!3 = !{!4, !"notcold"}
+!3 = !{!4, !"notcold", i64 100}
 !4 = !{i64 9086428284934609951, i64 -5964873800580613432, i64 2732490490862098848, i64 8632435727821051414}
-!5 = !{!6, !"cold"}
+!5 = !{!6, !"cold", i64 400}
```

Comment:
Is it possible to add another test case where the number of sizes listed is > 1?

Reply:
That isn't possible. Each MIB only has one aggregate size from the profile.